va/gsb.go

```diff
@@ -48,6 +48,12 @@ func (va *ValidationAuthorityImpl) isSafeDomain(ctx context.Context, domain stri
 	}
 
 	list, err := va.safeBrowsing.IsListed(ctx, domain)
+	if canceled.Is(err) {
```

Comment:
I think you forgot to import `github.com/letsencrypt/boulder/canceled`. This is failing in CI:
```
go vet ./...
# github.com/letsencrypt/boulder/va
va/gsb.go:51:5: undefined: canceled
```